docs/src/main/paradox/docs/releases/v1.4-to-v1.5-migration.md

```diff
@@ -0,0 +1,384 @@
+|Cassandra password                                       | app.database.cassandra.password              |                                                               |
+|Cassandra timeout                                        | datastax-java-driver.basic.request.timeout   | 10 seconds                                                    |
+|Secrets encryption password                              | app.encryption.password                      |                                                               |
+|Secrets encryption salt                                  | app.encryption.salt                          |                                                               |
+|Remote storage enabling                                  | plugins.storage.storages.remote-disk.enabled | true/false                                                    |
+|S3 storage enabling                                      | plugins.storage.storages.amazon.enabled      | true/false                                                    |
+|Delta 1.4 keyspace                                       | migration.replay.keyspace                    | delta                                                         |
+|Bucket of the first event                                | migration.replay.first-time-bucket           | YYYYMMDDTHH:MM (Same as the one on your Delta 1.4 deployment) |
+
+Start Delta 1.5 with this configuration.
+
+### Check the delta logs
+* The following messages indicating migration should appear at startup:
+
+        2021-05-05 14:34:42 INFO  ch.epfl.bluebrain.nexus.delta.Main - Plugins discovered: blazegraph-1.5.xxx, elasticsearch-1.5.xxx, storage-1.5.xxx, composite-views-1.5.xxx, archive-1.5.xxx
+        2021-05-05 14:34:46 INFO  ch.epfl.bluebrain.nexus.delta.Main - Starting Delta in migration mode
+
+If remote/S3 storage are enabled, these lines should appear too:
+
+        2021-05-05 14:35:00 INFO  c.e.b.n.d.p.s.StoragePluginConfig - Remote-disk storage is enabled
+
+Keep looking regularly in the logs to look if the migration is still ongoing without problems.
+If the migration crashes, a restart should allow it to start back from where it stopped.
+
+### Monitor the health of the platform
+Please check the general health of the platform (especially Cassandra available disk space).
+
+### Check the migration progress:
+Run the following query on table projections_progress in cqlsh (or your favorite cql client):
+
+        select projection_id, processed, failed, warnings, toTimestamp(offset) as migratedUpTo, timestamp from delta_1_5.projections_progress;
+
+This will return a similar response:
+
+        projection_id  | processed | failed | warnings | migratedupto
+        -----------+--------+----------+--------------------------------------------------
+        migration-v1.5 |     38499 |  10000 |       12 | 2019-04-10 08:53:01.482000+0000 
+
+The table contains the following information:
+* processed:      total number of processed events
```

Comment:
There is a wrong formatting of this. You need to add an extra empty line before starting the `*` block